data-prepper-plugins/opensearch/README.md

```diff
@@ -596,3 +596,20 @@ The default behavior is to process all indices.
 #### <a name="index_configuration">Index Configuration</a>
 
 * `index_name_regex`: A regex pattern to represent the index names for filtering
+
+## Developer guide
+
+### Integration tests
+
+#### Run OpenSearch
+
+Start an instance of OpenSearch that listens to opens port 9200 with default user admin:admin.
+```
+docker run -p 9200:9200 -e "discovery.type=single-node" opensearchproject/opensearch:1.0.0
```

Comment:
Suggest use `latest` tag

Reply:
I'd prefer to have something stable to be sure that the failed job is not because of the latest OpenSearch version, but it's not important to me either or.

Reply:
Feel free to merge https://github.com/opensearch-project/data-prepper/pull/4330/files#r1553510299